### PR TITLE
Missing Cherry-picks for 3.18

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: packages/python-django/python-django.spec [hunk 1]
```diff
@@ -6,8 +6,8 @@
 %global srcname django
 
 Name:           %{?scl_prefix}python-%{srcname}
-Version:        3.2.12
-Release:        2%{?dist}
+Version:        3.2.13
+Release:        1%{?dist}
```
Comment (evgeni) [inline packages/python-django/python-django.spec:10]: this release already exists: http://koji.katello.org/koji/buildinfo?buildID=70459

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/python-django/python-django.spec (hunk 1 above)
Comment (Odilhao) [reply packages/python-django/python-django.spec:10]: So here's the problem, If I cherry pick and later bump in the same PR, our rpmcheck will fail, let me cherry-pick the change, and after the merge I'll bump the release to build against 3.9

Comment (evgeni) [reply packages/python-django/python-django.spec:10]: Yeah, I guess we could merge this, which will skip the build of django, as it's already there, and then do another bump for the actual rebuild

**Review by evgeni — APPROVED** [review]

File: packages/python-django/python-django.spec (hunk 1 above)
Comment (Odilhao) [reply packages/python-django/python-django.spec:10]: Ack, and the cherry-pick was not clean from 3.17, because we did a bump later for python39

**Pull request merged**